tests/main/buildmode/task.yaml

```diff
@@ -20,3 +20,7 @@ execute: |
     for p in snapd snap-repair; do
         objdump -f /usr/lib/snapd/"$p" | MATCH "$needle"
     done
+
+    # C binaries are always built in non-PIE mode because we do not enable
+    # hardening
+    objdump -f /usr/lib/snapd/snap-confine | not MATCH 'DYNAMIC'
```

Comment:
Can you please add a comment that explains how absence of DYNAMIC is related to PIE.